grommet/grommet
# feat(Button): add transition to button theme

Signed-off-by: Lars Hisken <lars.hisken@bettyblocks.com>



#### What does this PR do?

It makes Button transitions configurable via the theme.

#### Where should the reviewer start?

At the StyledButton; and by testing the `button.transition` theme keys.

#### What testing has been done on this PR?

I tested it manually with Storybook.

#### How should this be manually tested?

Add the transition keys (as stated in the updated docs) to the Button's theme. Inspect the button and check the updated CSS transition keys.

#### Any background context you want to provide?

We're using Grommet in our product (with great success) and wish to theme it according to our style-guide. Animations are also part of that.

#### What are the relevant issues?

#3706

#### Screenshots (if appropriate)

#### Do the grommet docs need to be updated?

Yes.

#### Should this PR be mentioned in the release notes?

Not sure.

#### Is this change backwards compatible or is it a breaking change?

Backwards compatible.
